NEWS

```diff
@@ -1,3 +1,53 @@
+Overview of changes leading to 3.0.0
+Friday, September 17, 2021
+====================================
+- Unicode 14.0 support (David Corbett).
+- The hb-subset API and the harfbuzz-subset library's ABI are now declared
+  stable. The harfbuzz-subset library work was not possible without Garret's
+  and Qunxin's work of Google Fonts, and earlier work of Ariza from Adobe.
```

Comment:
One last nit. Let's do this professionally.
Garret Rieger. Qunxin Liu. Michiharu Ariza.